firefly_client/firefly_client.py

```diff
@@ -1453,6 +1454,64 @@ def rvstring_from_dict(self, rvdict):
                               gamma_value = rvdict['gamma_value'])
         return rvstring
 
+    # -----------------------------------------------------------------
+    # image line based footprint overlay
+    # -----------------------------------------------------------------
+    def overlay_imagepixel_based_footprint_layer(self, footprint_data, title=None,
+                                          footprint_layer_id=None, plot_id=None, **additional_params):
+        """
+        Overlay a footprint described in image line style on the loaded FITS images.
```

Comment:
"Overlay a footprint dictionary on displayed images.
The dictionary must be convertible to JSON format.